lib/v2/screens/profile_screens/security/security_screen.dart

```diff
@@ -70,17 +70,19 @@ class SecurityScreen extends StatelessWidget {
                         onTap: () => Share.share(settingsStorage.privateKey!),
                       ),
                       BlocBuilder<SecurityBloc, SecurityState>(
-                        buildWhen: (previous, current) => previous.hasNotification != current.hasNotification,
+                        buildWhen: (previous, current) =>
+                            previous.hasNotification != current.hasNotification ||
+                            previous.guardianStatusText != current.guardianStatusText,
```

Comment:
I know that this screen is difficult to handle the listeners because it was one of the first I did and at that time I still did not use the pageCommand.

I must come back here later and adjust these listeners